src/dbt_score/cli.py

```diff
@@ -93,8 +93,17 @@ def cli() -> None:
     is_flag=False,
     default=None,
 )
+@click.option(
+    "--show-all",
+    help="If set to True,show all models and all rules in output"
+    "when using `plain` as `--format`."
+    "Default behavior is to only show failing models and violated rules",
```

Comment:
```suggestion
    help="If set to True, show all models and all rules in output "
    "when using `plain` as `--format`. "
    "Default behavior is to only show failing models and violated rules.",
```

Reply:
To fix this:

![image](https://github.com/user-attachments/assets/24c62535-90b6-4023-83de-feab1444a9d6)